php/libraries/NDB_BVL_Instrument.class.inc

```diff
@@ -2086,7 +2086,11 @@ abstract class NDB_BVL_Instrument extends NDB_Page
      */
     function _nullScores(array $scoreCols): void
     {
-        $data = $this->loadInstanceData($this);
+        $record    = $this->loadInstanceData($this);
```

Comment:
Note: this will need to change to $this->getInstanceData() when merged back into main. (loadInstanceData is deprecated, but getInstanceData isn't on this branch..)